pydantic/_internal/_model_construction.py

```diff
@@ -108,7 +121,10 @@ def wrapped_model_post_init(self: BaseModel, __context: Any) -> None:
                     namespace['model_post_init'] = init_private_attributes
 
             namespace['__class_vars__'] = class_vars
-            namespace['__private_attributes__'] = {**base_private_attributes, **private_attributes}
+            namespace['__private_attributes__'] = {
+                **base_private_attributes,
+                **private_attributes,
+            }
```

Comment:
Were these formatting changes made by `ruff format`?

Reply:
they were made by `make format`

Reply:
Huh, when I run `make format`, it doesn't change those lines...